package.json

```diff
@@ -135,4 +135,4 @@
     "requiresKeyFile": true,
     "requiresProjectId": true
   }
-}
+}
```

Comment:
Looks good. Can you remove changes to package.json. You dont need to check this in.